tools/qemu_cloud_start

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+for id in `seq 0 10` ; do
+  #./tools/qemu_dev_start --no-serial --node-id $id $@ &
```

Comment:
This is commented on purpose?

Reply:
nope. sorry about that, anyway refactored everything into an ansible-playbook, much more powerful